src/components/Card.js

```diff
@@ -0,0 +1,15 @@
+import '../styles/card.css';
+import noneImg from '../assets/noneImg.png';
+function Card({ id, time, imgUrl = noneImg, title, description, date, url }) {
+  return (
+    <a className="card" key={id} href={url} target="_blank">
+      <img className="card__img" src={imgUrl} alt={title}></img>
```

Comment:
> 카드 컴포넌트에서 hover 상태일때 이미지가 확대만 되고 범위 밖으로 나가지 않게 하고 싶었는데 그렇게는 할 수 없을까요?

이렇게 div태그로 감싸고, div에서 overflow hidden을 주면 될 것 같습니다
```suggestion
      <div style={{ overflow: "hidden", width: "100%" }}>
        <img className="card__img" src={imgUrl} alt={title}></img>
      </div>
```